src/screens/SUE/SueReportScreen.tsx

```diff
@@ -374,6 +374,10 @@ export const SueReportScreen = ({
         if (isAnyInputMissing) {
           return texts.sue.report.alerts.missingAnyInput;
         }
+
+        if (selectedPosition) {
+          Alert.alert(texts.sue.report.alerts.hint, texts.sue.report.alerts.imageLocation);
```

Comment:
in my case the alert came too late when i switched to the next step:

<img width="379" alt="image" src="https://github.com/smart-village-solutions/smart-village-app-app/assets/1942953/c5c8ba5d-efb6-4b18-b5a8-48e3beeeb247">

Reply:
What exactly do you mean when you say it came late? You pressed the button and were directed to the next step. How many seconds afterwards did you get the warning? The fact that the alert comes after the continue button is pressed does not mean that it comes late. But if you receive the alert after 3-4 seconds, then there is a problem.

Reply:
in your screenshot the alert cam on step 2, where i upload the image. this is what i expect.
in my case i needed to go to the next step to get the alert, which is confusing.

Reply:
Yes, but the ticket says it's supposed to be like this. If you want to see this warning when the image is uploaded, we will see this warning every time the image is selected. I think it's more confusing this way.

Reply:
all right